app/src/main/java/org/mozilla/fenix/search/SearchFragment.kt

```diff
@@ -274,7 +294,7 @@ class SearchFragment : Fragment(), UserInteractionHandler {
         }
 
         if (!permissionDidUpdate) {
-            toolbarView.view.requestFocus()
+            toolbarView.view.edit.focus()
```

Comment:
Some minor cleanup. These weren't actually focusing on the url bar properly, so I added this in a few places we were doing `requestFocus()`